backend/app/routers/files.py

```diff
@@ -406,3 +404,17 @@ async def resubmit_file_extractions(
             FileOut(**file.dict()), rabbitmq_client, user, credentials
         )
     return resubmit_success_fail
+
+
+@router.post("/{file_id}/thumbnail/{thumbnail_id}", response_model=FileOut)
```

Comment:
i think we are just documenting one of the visualization bytes id as the thumbnail?

Reply:
Okay. Thanks. In that case, PUT may be a better operation here, as POST always requires one to create a new resource. If this endpoint is called twice, ideally, it should have created two resources, but with PUT, it can just replace the content and wouldn't change the thumbnail if it was called twice with the same URI. 

Please see for reference: https://restfulapi.net/rest-put-vs-post/

Reply:
Agree. or maybe Patch?

Reply:
Yes, PATCH seems fine for now. Thanks.